pkg/controller/controller.go

```diff
@@ -520,9 +527,9 @@ func (ctrl *resizeController) markPVCResizeInProgress(pvc *v1.PersistentVolumeCl
 	newPVC.Status.Conditions = util.MergeResizeConditionsOfPVC(newPVC.Status.Conditions,
 		[]v1.PersistentVolumeClaimCondition{progressCondition})
 
-	updatedPVC, err := ctrl.patchClaim(pvc, newPVC)
+	updatedPVC, err := ctrl.patchClaim(pvc, newPVC, true /* addResourceVersionCheck */)
 	if err != nil {
-		return nil, err
+		return updatedPVC, err
 	}
 	return updatedPVC, nil
```

Comment:
Since `updatedPVC` is always returned now, we can remove the "if err != nil" code block and just have "return updatedPVC, err" as this will cover the case when err is nil as well.

Reply:
Fixed.